### ISSUE TITLE
[BUG] [Custom Controls] Precision of `height` and `width` is lost after saving

### ISSUE THREAD

**Issue by Felix14-v2** [PojavLauncherTeam/PojavLauncher#4824]
### Describe the bug

When you import and edit custom controls, you can notice, that it slightly changes after saving even if no changes has been done in it. This occurs because the  `height` and `width` values of some buttons are slightly changes each time you save the controls. This is well noticeable with my controls, for example.

### The log file and images/videos

<details>
 <summary>Difference</summary>

![image](https://github.com/PojavLauncherTeam/PojavLauncher/assets/75726196/26e6fa07-cfe7-47ef-a65b-714413d158d0)
![image](https://github.com/PojavLauncherTeam/PojavLauncher/assets/75726196/f7a4ff1b-df4e-41ca-9a95-5944748f0efb)

Before saving:
![image](https://github.com/PojavLauncherTeam/PojavLauncher/assets/75726196/677fddfe-5e60-4435-acf1-255d0ec8bda0)
After saving:
![image](https://github.com/PojavLauncherTeam/PojavLauncher/assets/75726196/c5ad048f-0be0-4558-8a79-821157933691)

</details>

### Steps To Reproduce

1. You can create your own or just import [my controls](https://mediafire.com/file/ku0xj4jz4drw0lf)
2. Open it in the launcher editor
3. Save the controls
4. Open the saved file in a diff finder
5. See the difference
You can do these steps several times, and each time the saved values will be different.

### Expected Behavior

`width` and `height` values should be the same after saving as before it.

### Platform

```markdown
- Platform: Android
- CPU architecture: arm64
- Android version: 11
- PojavLauncher version: latest build
```

### Anything else?

It seems that this is a separate issue from #4706, because the corruption happens only after saving, while #4706 affects the entire scaling. But they may have the same root.

**Comment by Mathias-Boulay** [comment]
my man do be going out of spec each time and complaining about the result

**Comment by Mathias-Boulay** [comment]
On another note, feel free to propose a PR for this as I don't have much time available

**Comment by Felix14-v2** [comment]
> feel free to propose a PR

I would if I could, but I'm not a programmer yet 🥲
So all I can do to help in solving a problem is to provide a detailed bug report. I'm not insisting on fixing this, you have many other, more important tasks. I just have a small hope that this will be solved one day.

> and complaining about the result

This is not a complaint, I just found a bug ¯\\\_(ツ)\_/¯
Personally I don't edit my controls in the launcher editor and I have no issues with that, but... but other users of my controls often want to edit it (for example, to resize the hotbar buttons, because they can't use the global scaling for that due to 4706), and they can't do that. So this is not a complaint to you — you all are cool guys, you are doing a great project, and I try to help you with this in any way I can (working on high-quality localization, for example), and I just want users to be able to play comfortably :)

> do be going out of spec each time

Well, initially my controls was a proof of concept, but along with the growth of its functionality, more and more people wanted to get it. So I tried to workaround the technical problems from my side by using some tricks to minimize the negative effect, but... not everything is in my power. We need a little help here.
Anyways, thanks for your time! I highly appreciate your work.

**Comment by kedrick021** [comment]
> > feel free to propose a PR
> 
> I would if I could, but I'm not a programmer yet 🥲 So all I can do to help in solving a problem is to provide a detailed bug report. I'm not insisting on fixing this, you have many other, more important tasks. I just have a small hope that this will be solved one day.
> 
> > and complaining about the result
> 
> This is not a complaint, I just found a bug ¯\_(ツ)_/¯ Personally I don't edit my controls in the launcher editor and I have no issues with that, but... but other users of my controls often want to edit it (for example, to resize the hotbar buttons, because they cant use the global scaling for that due to 4706), and they can't do that. So this is not a complaint to you — you all are cool guys, you are doing a great project, and I try to help you with this in any way I can (working on high-quality localization, for example), and I just want users to be able to play comfortably :)
> 
> > do be going out of spec each time
> 
> Well, initially my controls was a proof of concept, but along with the growth of its functionality, more and more people wanted to get it. So I tried to workaround the technical problems from my side by using some tricks to minimize the negative effect, but... not everything is in my power. We need a little help here. Anyways, thanks for your time! I highly appreciate your work.

Can I use your control?

**Comment by Felix14-v2** [comment]
> Can I use your control?

Don't ask, just use it. But there are some technical limitations atm, so write to me in discord (felix14_v2) to get more details
